### PR TITLE
Fixing 1166 (Style guide - test cases - active voice)

### PR DESCRIPTION
**Pull request by TheDauntless**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1166 .

Apparently this was already in the intro of the style guide ("When giving technical instructions, address the reader in the second person."), but I've added the examples from #1166 and added some more consistency in the title numbering.

### CODE REVIEW

File: style_guide.md [hunk 1]
```diff
@@ -148,9 +148,19 @@ Spell out abbreviations the first time they are used.
...
+### Addressing the reader in test cases
+
+Throughout the guide, you may want to address the reader in order to tell him what to do, or what he should notice. For any such case, use an active approach and simply address the reader using "you".
+
+**Correct:** If you open the AndroidManifest.xml file, you will see a main Application tag, with the following attributes: atr1, atr2 and atr3. If you run the following command, you will see that atr1 is actually dangerous: ...
+
+**Wrong:** The AndroidManifest.xml file contains an Application tag, with the following attributes: atr1, atr2 and atr3. The command below shows that atr1 is dangerous: ...
+
+**Wrong:** If we open the AndroidManifest.xml file, we will see a main Application tag, with the following attributes: atr1, atr2 and atr3. If we run the following command, we will see that atr1 is actually dangerous: ...
+
```
Comment (sushi2k) [inline style_guide.md]: That's fine for me. @commjoen You also ok with it?

**Comment by TheDauntless** [comment]
Since I had this one open, I've added the style guide for the API level references.

File: style_guide.md [hunk 2]
```diff
@@ -24,11 +24,7 @@ Consider creating a supporting document and linking to it from the page rather t
 
 Keeping accurate and timely content establishes the 'Mobile Security Testing Guide' as a credible and trustworthy source of information.
 
-Update pages in order to keep information, such as data and statistics, timely and accurate.
```
Comment (TheDauntless) [inline style_guide.md]: (This feels like exactly the same as the next phrase, so I deleted it)

File: style_guide.md (hunk 1 above)
Comment (commjoen) [reply style_guide.md]: 👍

**Pull request merged**